src/Mod/Image/ImageTools/_CommandImageScaling.py

```diff
@@ -168,10 +157,10 @@ def accept(self):
                 self.dialog.hide()
                 FreeCADGui.SendMsgToActiveView("ViewFit")
             except (ValueError, ZeroDivisionError):
-                self.label1.setText(_translate("Dialog", "<font color='red'>Enter distance</font>", None))
+                self.label1.setText(QtGui.QApplication.translate("Dialog", "<font color='red'>Enter distance</font>", None))
```

Comment:
The text to be translated is `Enter distance` and the rest is technical stuff (`<font color='red'>` or `</font>`) that shouldn't be part of the expression. This is because there is a certain chance that a translator makes a mistake and thus the dialog will show garbage.